src/app/pages/session/edit/[id]/index.vue

```diff
@@ -17,19 +16,73 @@
               })
             "
           >
-            <IHeroiconsIdentification />
+            <IMaevsiPerson class="h-6 w-6" :alt="t('iconAltPerson')" />
           </CardButton>
-          <CardButton :title="t('signOut')" @click="signOut">
-            <IHeroiconsOutlineLogout />
-            <template #iconSecondary />
+          <CardButton
+            :title="t('contactBook')"
+            :to="
+              localePath({
+                name: 'contact',
+                params: {
+                  username: store.signedInUsername,
+                },
```

Comment:
The contact book path (`/contact`) doesn't have a username parameter, this part can be removed.